tools/reviewer/web/README.md

```diff
@@ -2,7 +2,8 @@
 Run `ng install` for the first time.
 
 Run `npm run protoc` to generate proto functions.
-You need installed google-protobuf proto3 on your computer.
+You need installed google-protobuf version 3.5 or above on your computer.
+Supported on Linux and Mac machines only.
```

Comment:
How about we mention Linux & Mac support  at the bottom?

Reply:
Well, most of the app supports windows. Only `npm run protoc` doesn't. For example, if I have generated proto functions, I can continue development on windows without any problems.

Reply:
Got it.